### DISCUSSION TITLE
[feat] cors plugin add a config field: override_origin

### DISCUSSION THREAD

**Discussion by WALL-E** [Kong/kong#7261] [Ideas and feature requests]
A boolean value that instructs the plugin to set the config.origins from request headers "Origin" as the value.

The user has the ability to turn off this feature.

https://github.com/Mashape/kong/blob/master/kong/plugins/cors/handler.lua#L69L82

**Comment by p0pr0ck5** [comment]
@WALL-E would you consider submitting a PR for this functionality?

**Comment by WALL-E** [comment]
@p0pr0ck5 OK
